modules/migration-rolling-back-migration-cli.adoc

```diff
@@ -3,15 +3,54 @@
 // * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
 // * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
 
-[id='migration-manually-rolling-back-migration_{context}']
-= Manually rolling back a migration
+[id='migration-rolling-back-migration-cli_{context}']
+== Rolling back a migration from the CLI
 
-If your application was stopped during a failed migration with the {mtc-full} ({mtc-short}), you must roll it back manually in order to prevent data corruption in the persistent volume.
+You can roll back a migration from the CLI.
 
-This procedure is not required if the application was not stopped during migration, because the original application is still running on the source cluster.
+If your application was stopped during a failed migration, you must roll back the migration in order to prevent data corruption in the persistent volume.
+
+Rollback is not required if the application was not stopped during migration because the original application is still running on the source cluster.
 
 .Procedure
 
+ifeval::["{mtc-version}" >= "1.4"]
```

Comment:
I think it would be better to apply the changes to *just* the versions that they apply to instead of adding these version-based conditionals.

Reply:
MTC versions do not align with OCP release schedule. Unless you have a way of ensuring that the 1.4 functionality will only appear on Jan. 14, I do not see an alternative.

Reply:
Split the PR and send a merge request to the channel on 1/14. @abrennan89 also works out of phase from core OCP and stages content for review without a merge request, then sends merge requests whenever release actually happens. @vikram-redhat, is there something that we can do to make out-of-sync PRs go in more smoothly?

Reply:
We have a lot of content that goes in out of sync, and this is no different. No reason this couldn't flow the same way (and others). Just send a merge request on the day itself. You don't need someone within the peer review squad to merge, anyone with merge rights in the team can merge (as long as QE/peer review has been received).

Reply:
OK